model.md

```diff
@@ -0,0 +1,164 @@
+# Model
+
+### <a name="struct"></a> Struct
+
+This is the primary object...
+
+| Field   | Type                      | Required | Notes                                                  |
+|---------|---------------------------|----------|--------------------------------------------------------|
+| name    | ref                       | true     | The common name used to identity the type of structure |
+| label   | string or [Label](#label) | true     | Describes the struct in the UI                         |
```

Comment:
If we have types that vary like this in our schema definition, should we also allow that to be represented?  Then we can use the resultant UI generator to edit the schemas.